app/controllers/activities_controller.rb

```diff
@@ -1,6 +1,8 @@
 class ActivitiesController < ApplicationController
+  before_action :authenticate_user!
+
  def index
- 	sleep 0.7
+    sleep 0.7
     @activities = PublicActivity::Activity.order("created_at desc").where(recipient_id: current_user).where("owner_id not in (?)", current_user).all
   end
 end
```

Comment:
Leave it. It is not that big a deal. I can't think of situation where some user will be see other's activity, as you had pointed out that we were using current_user to find the activity.